ui/components/institutional/signature-mismatch-banner/signature-mismatch-banner.tsx

```diff
@@ -55,26 +46,12 @@ const MMISignatureMismatchBanner: React.FC = memo(() => {
     return null;
   }
 
-  return (
-    <Box
-      display={Display.Flex}
-      width={BlockSize.Full}
-      padding={4}
-      backgroundColor={BackgroundColor.primaryMuted}
-    >
-      <Icon
-        name={IconName.Info}
-        color={IconColor.infoDefault}
-        marginRight={2}
-      />
-      <Text variant={TextVariant.bodyXs} color={TextColor.textDefault}>
-        {t('mismatchAccount', [
-          shortenAddress(selectedAccount?.address),
-          shortenAddress(fromAccount?.address),
-        ])}
-      </Text>
-    </Box>
-  );
+  const message = t('mismatchAccount', [
+    shortenAddress(selectedAccount?.address),
+    shortenAddress(fromAccount?.address),
+  ]);
+
+  return <BannerAlert marginTop={3} title={message} />;
```

Comment:
```suggestion
  return <BannerAlert title={message} />;
```
hmm so this without the other change would result in
![CleanShot 2024-07-02 at 18 54 28](https://github.com/MetaMask/metamask-extension/assets/20778143/78c70aba-4d79-4735-89a3-8f13f25f321f)

which lays over a white background rather than a gray background. If you'd like, we could revert this and merge in the code (now that we're here and I've had a second thought, I think this is better to have the style outside :) 

then myself or someone else can create another PR to move and wrap the BannerAlert in `ui/pages/confirmations/confirm/confirm.tsx`